infrastructure/resource_group_level/modules/webapps/portal-api.bicep

```diff
@@ -12,9 +12,9 @@ param dtfsCentralApiHostname string
 param tfmApiHostname string
```

Comment:
I see GitHub has realised this is a rename and edit. Not sure why it hasn't for some of the others.